requirements.txt

```diff
@@ -1,2 +1,11 @@
-cryptography==2.8
-pandas==0.25.3
\ No newline at end of file
+certifi==2020.6.20
+chardet==3.0.4
+idna==2.10
+numpy==1.19.0
+pandas==1.0.5
+pyEzviz==0.1.5.7
```

Comment:
Why is pyEzviz part of the requirements?

Reply:
I'll update this file. Do you know if this file has any practical uses?

Reply:
Removed pyEzviz from requirements file.

Reply:
This file lets you install all dependencies for this project in one go:
```
pip3 install -r requirements.txt
```